mkdocs/docs/contributing/newk8sstack.md

```diff
@@ -0,0 +1,125 @@
+---
+template: main.html
+---
+
+# Add a K8s Stack / Service Mesh / Ingress
+
+Performing Iter8 experiments requires RBAC rules, which are contained in [this Kustomize folder](https://github.com/iter8-tools/iter8/tree/master/install/core/rbac/stacks) and are installed as part of Iter8 installation.
+
+Enable Iter8 experiments over a new K8s stack by extending these RBAC rules.
+
+## Step 0: Fork Iter8
+Fork the [Iter8 GitHub repo](https://github.com/iter8-tools/iter8). Locally clone your forked repo.
+
+For the rest of this document, `$ITER8` will refer to the root of your local Iter8 repo.
+
+## Step 1: Edit `kustomization.yaml`
+```shell
+cd $ITER8/install/core/rbac/stacks
+```
+
+Edit `kustomization.yaml` to add your K8s stack. At the time of writing, it contains the following stacks:
+```yaml
+resources:
+- iter8-knative
+- iter8-istio
+- iter8-kfserving
+# -iter8-<your stack> # add your stack here
+```
+
+## Step 2: Create subfolder
+```shell
+mkdir iter8-<your stack>
+cp iter8-kfserving/kustomization.yaml iter8-<your stack>/kustomization.yaml
+```
+
+## Step 3: Create RBAC rules
+```shell
+cd iter8-<your stack>
+```
+
+=== "Foo resource & Istio virtual service"
+    Suppose your stack defines a custom resource called `foo` and uses the Istio service mesh.
+    Create RBAC rules that will enable Iter8 to manipulate `foo` resources and Istio virtual service resources during experiments. You can do so by creating `roles.yaml` and `rolebindings.yaml` files as follows.
```

Comment:
This implies the user may have custom resources. It doesn't matter that they are custom. Any needed resource. Maybe just "suppose your stack uses resources of type `foo` and the Istio service mesh..."

I wonder if it would be useful to help identify the kinds of resources that should be included in such rules. That is, any resources that will appear in a weightObjRef in versionInfo .

Reply:
Done.